pom.xml

```diff
@@ -147,7 +147,9 @@
             <extensions>true</extensions>
             <configuration>
               <serverId>ossrh</serverId>
-              <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+              <nexusUrl>https://google.oss.sonatype.org/</nexusUrl>
+              <autoReleaseAfterClose>false</autoReleaseAfterClose>
```

Comment:
Quick question: is the default true? It is also true for `https://oss.sonatype.org/`?

Reply:
Good call, it is false by default. I will omit it to keep the configuration shorter.

Settings copied from here: https://github.com/googleapis/java-shared-config/blob/master/pom.xml#L81